app/scripts/views/mixins/flow-events-mixin.js

```diff
@@ -9,6 +9,8 @@ define(function (require, exports, module) {
 
   const Flow = require('models/flow');
 
+  const TAB_KEY = 9;
```

Comment:
Can you use the one from [lib/key-codes](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/key-codes.js#L21)?